src/Utilities/FlowAnalysis/FlowAnalysis/Analysis/TaintedDataAnalysis/TaintedDataAnalysis.TaintedDataOperationVisitor.cs

```diff
@@ -232,6 +232,15 @@ public override TaintedDataAbstractValue VisitObjectCreation(IObjectCreationOper
                 {
                     return TaintedDataAbstractValue.CreateTainted(method, originalOperation.Syntax, this.OwningSymbol);
                 }
+                else if (this.IsSanitizingInstanceMethod(method))
+                {
+                    if (AnalysisEntityFactory.TryCreate(visitedInstance, out AnalysisEntity analysisEntity))
```

Comment:
`visitedInstance` might be null for static method invocations. Are SanitizingInstanceMethods guaranteed to be non-static? Otherwise, consider adding a null check for `visitedInstance`.